packages/ra-ui-materialui/src/detail/Tab.tsx

```diff
@@ -132,6 +134,7 @@ Tab.propTypes = {
     icon: PropTypes.element,
     label: PropTypes.string.isRequired,
     path: PropTypes.string,
+    spacing: any,
```

Comment:
Why not `PropTypes.any`?

Reply:
The question is, really: how did it compile?

Fixed in the next commit.